templates/cassiopeia/scss/vendor/bootstrap/_custom-forms.scss

```diff
@@ -35,27 +35,23 @@
     }
   }
 
-  // All class names are duplicated custom select and form select for b/c with bootstrap 4
-  &.custom-select-color-state, &.form-select-color-state {
+  &.form-select-success {
```

Comment:
I thought we were leaving in the b/c mapping layer for 3rd parties? Looks like you removed them in the conflicts

Reply:
Apologies, I'm getting a bit confused here with some of the comments. These classes are of little use without https://github.com/wilsonge/joomla-cms/pull/55#discussion_r559739569

I've re-added them with https://github.com/wilsonge/joomla-cms/pull/55/commits/57f096b74a9eac347843a19789d974f4f4aadf23

Also moved the extended `.custom-select` as it needs to be loaded before the custom CSS. 

Tested both `form-select` and `custom-select` and both function correctly.